schema205/schema_tables.py

```diff
@@ -0,0 +1,489 @@
+            "data_path": ["Object Type"],
+            "equals": "String Type",
+        },
+        "columns": [
+            {
+                "data_path": [],
+                "name_in_table": "Data Type",
+            },
+            {
+                "data_path": ["Description"],
+                "name_in_table": "Description",
+            },
+            {
+                "data_path": ["JSON Schema Pattern"],
+                "name_in_table": "JSON Schema Pattern",
+                "default": "(Not applicable)",
+                "transforms": [
+                    tables.add_space_between("-", "["),
+                    tables.add_space_between("]", "]"),
+                    tables.add_space_between(")", ")"),
+                    tables.add_space_between(")", "("),
+                    tables.add_space_between(")", "\\."),
+                    tables.add_space_between("*", "["),
+                    tables.add_space_between("0", "|"),
+                    tables.add_space_between("*", "|"),
+                    tables.add_space_between("+", "("),
+                    tables.add_space_between("?", "("),
+                    tables.add_space_between("]", "["),
+                    tables.add_space_between("|", "["),
+                    tables.add_space_between("]", "|"),
+                    tables.add_space_between("]", "{"),
+                    tables.add_space_between("}", ":"),
+                    tables.add_space_between("]", ":"),
+                    tables.add_space_between(")", ":"),
+                    tables.add_space_between(":", "["),
+                    tables.add_space_between(":", "("),
+                    tables.add_space_between(":", "{"),
+                ],
+            },
+            {
+                "data_path": ["Examples"],
+                "name_in_table": "Examples",
+                "transforms": [
+                    tables.join_with_comma,
+                ],
+            }
+        ]
+    }
+
+
+def make_enumerator_table_spec(ident, yaml_src, caption, columns, key=None):
+    """
+    Make an Enumerator TableSpec
+    - ident: string, the id to select
+    - yaml_src: string, the path to the YAML file
+    - caption: string, the caption
+    - columns: array of column specs
+    - key: None or String or "Enumerators", the key to use to match on
+    """
+    if key is None:
+        key = "Enumerators"
+    def make_rows(_, obj):
+        enumerators = obj[key]
+        return [(k, enumerators[k]) for k in sorted(enumerators.keys())]
+    return {
+        "yaml_source": yaml_src,
+        "caption": caption,
+        "select_by": {
+            "data_path": [],
+            "equals": ident,
+        },
+        "derive_rows": make_rows,
+        "columns": columns,
+    }
+
+
+def name_to_yaml_path(name):
+    """
+    - name: string, name of a schema file
+    RETURN: string, path to YAML file
+    """
+    return os.path.join(SCHEMA_DIR, f'{name}.schema.yaml')
+
+
+def add_table(
```

Comment:
Is this old? It seems the same function is defined in another file. If this one is used, it needs to be cleaned up (RS_ID and AHRAE205 are not table types).